File: src/main/resources/output/IFC4X3_python/Entities/IfcStructuralConnectionCondition/IfcSlippageConnectionCondition/IfcSlippageConnectionCondition.py
class IfcSlippageConnectionCondition(IfcStructuralConnectionCondition):
    def __init__(self):
        self.SlippageX: IfcLengthMeasure = None
        self.SlippageY: IfcLengthMeasure = None
        self.SlippageZ: IfcLengthMeasure = None

File: src/main/resources/output/IFC4X3_js/Entities/IfcStructuralConnectionCondition/IfcSlippageConnectionCondition/IfcSlippageConnectionCondition.js
class IfcSlippageConnectionCondition extends IfcStructuralConnectionCondition {
    constructor() {
        /** @type {IFCLENGTHMEASURE} */
        this.SlippageX = null;
        /** @type {IFCLENGTHMEASURE} */
        this.SlippageY = null;
        /** @type {IFCLENGTHMEASURE} */
        this.SlippageZ = null;
    }
}
